addons/web/controllers/main.py

```diff
@@ -1668,7 +1668,7 @@ def get_fonts(self, fontname=None):
         else:
             current_dir = os.path.dirname(os.path.abspath(__file__))
             fonts_directory = os.path.join(current_dir, '..', 'static', 'src', 'fonts', 'sign')
-            font_filenames = sorted(os.listdir(fonts_directory))
+            font_filenames = sorted(glob.glob('%s/*.*[ot][tf]*' % fonts_directory))
```

Comment:
Glob seems way over-complicated, why not just filter the result of `os.listdir` via `str.endswith`?

Also is it really necessary to go and filter this? It's a folder that's part of Odoo and specifically for sign, surely we only put the fonts we want in there?

Reply:
I agree that by making the glob works for `woff`and `woff2`, it became ugly.
And yes we need to filter thath if we want to put the license files alongside the font file.
I'm sure that's what we want to facilitate the search. Another idea would be to use a `licenses` directory or file à la Debian.
My first approach was to use the Debian/copyright file... but it's not a good idea as the other packages would not benefit from that file.

Reply:
> And yes we need to filter that if we want to put the license files alongside the font file.

Ah that's the part I had not noticed.